frontend/src/components/ProfileCard/ProfileCard.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const ProfileCard = () => {
+  return (
+    <div className="card shadow-lg" style={{ width: "300px", margin: "auto", padding: "2px" }}>
+      <div className="card-body">
+        
+        {/* Flex container to align photo, score, and details */}
+        <div className="d-flex mb-3">
+          {/* Photo and Score Container */}
+          <div className="d-flex flex-column align-items-center">
+            {/* Photo Box */}
+            <div
+              className="d-flex justify-content-center align-items-center border mb-2"
+              style={{ width: "60px", height: "60px", borderRadius: "50%", marginBottom: "2px" }}
+            >
+              <p className="text-muted">Photo</p>
+            </div>
+
+            {/* Score Section */}
+            <div
+              className="border text-center p-1"
+              style={{ width: "90px", height: "40px" }}
```

Comment:
provide this via classes

Reply:
Completed this change in commit [3d7b685] . Moved the inline styles to the CSS class.